napari/layers/shapes/_shapes_key_bindings.py

```diff
@@ -54,43 +57,47 @@ def register_shapes_action(description):
     return register_layer_action(Shapes, description)
 
 
-@register_shapes_action(trans._('Add rectangles'))
+def register_shape_mode_action(description):
+    return register_layer_attr_action(Shapes, description, 'mode')
+
+
+@register_shape_mode_action(trans._('Add rectangles'))
```

Comment:
For consistency, we should stick with `shapes`, rather than `shape`. Sorry for the nitpicking on the names @Czaki :P